softgroup/basis/common-router/pom.xml

```diff
@@ -3,13 +3,17 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<parent>
+    <modules>
+        <module>router-api</module>
+		<module>router-impl</module>
+	</modules>
+    <parent>
 		<groupId>com.softgroup</groupId>
 		<artifactId>basis</artifactId>
```

Comment:
@Dominys but this is softgroup/basis/common-router/pom.xml . this pom.xml is for common-router module